### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Yehorkk**
- [ x ] Yellow stars are added with container modifier + pseudo-selector (NO extra classes)
- [ x ] Each BEM block has its own separate file
- [ x ] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [ x ] Code follows all the [Code Style Rules ❗️](./checklist.md)

- [DEMO LINK](https://yehorkk.github.io/layout_stars/)
- [TEST REPORT LINK](https://yehorkk.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/blocks/stars.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+
```
Comment (VitaliyBodnar) [inline src/blocks/stars.css:3]:
```suggestion
```

File: src/blocks/stars.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(../images/star.svg);
+  width: 16px;
```
Comment (VitaliyBodnar) [inline src/blocks/stars.css:9]: size 16px is duplicated, so it can be moved to the star-size variable

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]
Almost done!

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**